src/Message.vue

```diff
@@ -110,7 +110,7 @@ export default {
       return this.user && this.user.name
     },
     chatImageUrl() {
-      return (this.user && this.user.imageUrl) || chatIcon
+      return (this.user && this.user.imageUrl) || chatIcon || ''
```

Comment:
`chatIcon` should always be available as an asset, so a fallback for it does not really make sense. If it cannot be fetched, then it's either an issue on the user's side, or a bad deployment.

Reply:
I agree!